src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -433,20 +452,25 @@
 	top: 0;
 	bottom: 0;
 	pointer-events: none;
-	opacity: 0; /* hidden initially */
+	opacity: 0;
+	/* hidden initially */
```

Comment:
Undoing these formatting changes would be good to simplify the PR

Reply:
reverted all of those